packages/compose/src/with-global-events/index.js

```diff
@@ -61,7 +61,12 @@ function withGlobalEvents( eventTypesToHandlers ) {
 			}
 
 			render() {
-				return <WrappedComponent { ...this.props } ref={ this.handleRef } />;
+				return (
+					<WrappedComponent
+						{ ...omit( this.props, [ 'forwardedRef' ] ) }
```

Comment:
Performance-wise, we could probably benefit by just passing the original props as a standalone prop value from within `forwardRef`:

```js
 		return forwardRef( ( props, ref ) => {
 			return <Wrapper ownProps={ props } forwardedRef={ ref } />;
 		} );
```

For example, we do this with `withSelect`:

https://github.com/WordPress/gutenberg/blob/4fd5c20af4a8a46c92048574e363b07977aa0f3f/packages/data/src/components/with-select/index.js#L127